scripts/release/release_github.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+####
+# Utils
+####
+source ${BASH_SOURCE%/*}/utils.sh
+
+function tag_repository()
+{
+  VERSION=$1
+  git tag -a $VERSION -m "Release $VERSION"
+  git push origin --tags
+}
+
+function remove_tag()
+{
+  VERSION=$1
+  git tag -d $VERSION
+  git push --delete origin $VERSION
+}
+
+function generate_github_release_notes_post_data()
+{
+  cat <<EOF
+{
+  "tag_name":"$TAG_NAME",
+  "previous_tag_name":"$PREV_TAG_NAME",
+  "name":"$RELEASE_NAME",
+  "draft":false,
+  "prerelease":false,
+  "generate_release_notes":false
+}
+EOF
+}
+
+function generate_github_release_post_data()
+{
+  cat <<EOF
+{
+  "tag_name":"$TAG_NAME",
+  "name":"$RELEASE_NAME",
+  "body":"$DESCRIPTION",
+  "draft":false,
+  "prerelease":false,
+  "generate_release_notes":false
+}
+EOF
+}
+
+function generate_github_release_notes()
+{
+  SECRET=$1
+  curl --silent \
+    -X POST \
+    -H "Accept: application/vnd.github+json" \
+    -H "Authorization: Bearer $SECRET" \
+    https://api.github.com/repos/opentensor/bittensor/releases/generate-notes \
```

Comment:
Typically the generated notes would require some post-editing, so is the plan just to edit the release notes afterward directly?